Wed Oct  4 08:31:58 EDT 2006  Josef Svenningsson <josef.svenningsson@gmail.com>
  * Look for Text.Regex in package regex-compat. Needed for GHC 6.6
diff -rN -u old-hsdarcs/configure.ac new-hsdarcs-1/configure.ac
--- configure.ac	2007-01-22 02:16:07.000000000 -0500
+++ configure.ac	2007-01-22 02:16:08.000000000 -0500
@@ -110,6 +110,7 @@
 dnl Look for Text.Regex
 
 GHC_CHECK_MODULE(Text.Regex( mkRegex, matchRegex, Regex ), text, mkRegex undefined)
+GHC_CHECK_MODULE(Text.Regex( mkRegex, matchRegex, Regex ), regex-compat, mkRegex undefined)
 
 dnl See if we need a package for QuickCheck
 

Tue Aug 15 19:57:14 EDT 2006  Esa Ilari Vuokko <ei@vuokko.info>
  * Link to relevant symbol when checking for Control.Monad.Error
diff -rN -u old-hsdarcs/configure.ac new-hsdarcs-1/configure.ac
--- configure.ac	2007-01-22 02:26:47.000000000 -0500
+++ configure.ac	2007-01-22 02:26:49.000000000 -0500
@@ -117,8 +117,8 @@
 
 dnl See if we need the util or mtl packages for Control.Monad
 
-GHC_CHECK_MODULE(Control.Monad.Error, util, putStr undefined)
-GHC_CHECK_MODULE(Control.Monad.Error, mtl, putStr undefined)
+GHC_CHECK_MODULE(Control.Monad.Error, util, strMsg "foo" :: String)
+GHC_CHECK_MODULE(Control.Monad.Error, mtl, strMsg "foo" :: String)
 
 dnl See if we need a package for parsec...
 

Tue Aug 15 19:57:39 EDT 2006  Esa Ilari Vuokko <ei@vuokko.info>
  * Check for module Text.Html in package html
diff -rN -u old-hsdarcs/configure.ac new-hsdarcs-1/configure.ac
--- configure.ac	2007-01-22 02:25:55.000000000 -0500
+++ configure.ac	2007-01-22 02:25:57.000000000 -0500
@@ -124,6 +124,10 @@
 
 GHC_CHECK_MODULE(Text.ParserCombinators.Parsec, parsec, errorPos undefined)
 
+dnl Check if we need package html
+
+GHC_CHECK_MODULE(Text.Html, html, text "foo")
+
 dnl Deal with systems on which getCurrentDirectory uses '\\' rather than '/':
 
 WORKAROUND_getCurrentDirectory

